thread/common/omrthread.c

```diff
@@ -3307,7 +3321,9 @@ omrthread_park(int64_t millis, intptr_t nanos)
 	ASSERT(self);
 
 #if defined(OMR_THR_YIELD_ALG)
-	omrthread_park_spin(self, millis, nanos, &sleptDuration);
+	if (self->library->cpuUtilCache >= self->library->parkSleepCpuUtilThreshold) {
```

Comment:
This expression evaluates to `0 > 80` based on the default values, which will always be `true` and unconditionally invoke `omrthread_park_spin`. That doesn’t seem correct. There needs to be an explanation of how this heuristic is expected to change dynamically, as well as clearer documentation for OMR users on how to configure or utilize it.